changes/6810.fixed

```diff
@@ -0,0 +1 @@
+Fixed Bulk Edit Objects job when adding/removing a single tag to multiple objects.
```

Comment:
I'm *guessing* this isn't specific to tags, but any m2m with `add_` and `remove_` bulk edit fields? Agreed tags are the most likely scenario though, just wondering if the changelog should call out other cases as well?

Reply:
There are a lot of fields to call out lmk if this works. https://github.com/nautobot/nautobot/pull/6824/commits/55cf8d8f43f4a7d796a47329bc8ca00650301afd